skbio/stats/distance/_base.py

```diff
@@ -715,28 +767,31 @@ def from_iterable(cls, iterable, metric, key=None, keys=None):
 
         Notes
         -----
-        Symmetry and hollowness are assumed when calculating the distances via
-        `metric`. Therefore, distances are only computed for the strictly
+        When `validate=True`, distances are only computed for the strictly
```

Comment:
I can't comment on the lines above this diff, but can you remove "all" from the "Returns" section text? i.e. "The `metric` applied to pairwise elements in the `iterable`."